data/darktableconfig.xml.in

```diff
@@ -102,6 +102,13 @@
     <shortdescription></shortdescription>
     <longdescription></longdescription>
   </dtconfig>
+  <dtconfig>
+    <name>storage/piwigo/overwrite</name>
+    <type>int</type>
```

Comment:
Probably better using `boolean` here and `false` as default below.

Reply:
hm, file-on-disk is int as well, and we could easily extend the options (e.g. only update metadata, ask, ...)